portal/gil/static/js/lived_experience.js

```diff
@@ -17,9 +17,6 @@ $(document).ready(function() {
       });
     }
     if ($("main").attr("data-section") === "livedexperience") {
-      $("#lnReadMoreStory").addClass("disabled").on("click", function(e) {
-          e.preventDefault();
-          return false;
-      });
+      $("#lnReadMoreStory").hide();
```

Comment:
Why not hide this element with CSS? Seems like it would be quicker/less finicky

Reply:
ivan, sorry, didn't see this comment until now.  I will just make another PR for this.  agree.